alvr/client_core/src/connection.rs

```diff
@@ -203,6 +204,8 @@ fn connection_pipeline(
     let settings = stream_config.settings;
     let negotiated_config = stream_config.negotiated_config;
 
+    *ctx.velocities_multiplier.write() = settings.extra.velocities_multiplier;
```

Comment:
This doesn't actually update dynamically or am I missing something?

Reply:
nope, it doesn't update automatically. also it's not a slider. I don't think we should put any more effort into this. this is just for debug

Reply:
Fair, I just don't want it to confusing when someone's modifying it for debugging purposes and they don't know that it doesn't update dynamically. So probably put a comment onto the setting

Reply:
done, but i think that was unnecessary because the user can see it doesn't have the green dot

Reply:
But it's also not marked as `steamv-restart`, so yknow